app/migrations/Version20151221084137.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20151221084137 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_product_variant ADD available_to DATETIME DEFAULT NULL');
```

Comment:
You can easily `git reset --soft HEAD~1` one or two times to get back to your commit for available_to, then `git add . --all` then `git stash` your working directory, checkout to master, now delete `feature/browse-products-via-archetypes`, then `git checkout -b feature/browse-products-via-archetypes` to create a new branch **off of master**, then `git stash pop` now you have your working directory ready to commit your changes, then `git push origin feature/browse-products-via-archetypes --force`.

I'm sure there may be other easier ways but this is what comes of my mind quick ;)